vendor/k8s.io/kubernetes/pkg/securitycontextconstraints/capabilities/mustrunas.go

```diff
@@ -101,9 +101,21 @@ func (s *defaultCapabilities) Validate(pod *api.Pod, container *api.Container) f
 		return allErrs
 	}
 
+	allowedAdd := makeCapSet(s.allowedCaps)
+	allowAllCaps := allowedAdd.Has(string(api.CapabilityAll))
+	if allowAllCaps {
+		if len(s.requiredDropCapabilities) > 0 {
```

Comment:
If anything, that validation should be in the `New...` method since it is validation against the invariants.  Assuming you're trying to protect against misconfiguration of the parameters.  Can that even happen?  Do we need a migration of an invalid strategy?